docker/AcceptanceTests/start_acceptance.sh

```diff
@@ -26,7 +26,7 @@ sed -i -e "s/{ADM_USERNAME}/admin-${Region1}/" conf/settings.json
 sed -i -e "s/{ADM_TENANT_NAME}/admin-${Region1}/" conf/settings.json
 sed -i -e "s/{ADM_PASSWORD}/admin-${Region1}/" conf/settings.json
 
-host="fiwareaikos"
+host="fiware-aiakos"
```

Comment:
the configuration files for acceptance tests. Anyway, have you tested the acceptance tests?